apa-fr-provost.csl

```diff
@@ -174,36 +174,60 @@
       <else-if type="report">
         <text variable="title" font-style="italic"/>
         <group prefix=" (" suffix=")" delimiter=" ">
-          <text variable="genre"/>
-          <text variable="number" prefix="No. "/>
+          <choose>
+            <if variable="number" match="any">
+              <text value="Rapport" suffix=" "/>
+              <text variable="number" prefix="No. "/>
+            </if>
+          </choose>
         </group>
```

Comment:
@fharnois, you should be able to change

```xml
        <group prefix=" (" suffix=")" delimiter=" ">
          <choose>
            <if variable="number" match="any">
              <text value="Rapport" suffix=" "/>
              <text variable="number" prefix="No. "/>
            </if>
          </choose>
        </group>
```

to the simpler

```xml
        <group prefix=" (" suffix=")" delimiter=" ">
          <text value="Rapport"/>
          <text variable="number" prefix="No. "/>
        </group>
```

(groups automatically suppress text values if all called variables are empty; see http://docs.citationstyles.org/en/stable/specification.html#group)